timeline-chart/src/layer/time-graph-axis.ts

```diff
@@ -17,17 +17,21 @@ export class TimeGraphAxis extends TimeGraphLayer {
     private _keyUpHandler: { (event: KeyboardEvent): void; (this: Document, ev: KeyboardEvent): any; };
     private _keyDownHandler: { (event: KeyboardEvent): void; (this: Document, ev: KeyboardEvent): any; };
 
-    constructor(id: string, protected style?: { color?: number, lineColor?: number }) {
+    constructor(id: string, protected style?: { color?: number, lineColor?: number, verticalLinePosition?: VerticalLinePosition }) {
```

Comment:
To me it seems odd that the external user of TimeGraphAxis needs to import the type used in the constructor from an internal component. Seems like it should be defined in this class.

Reply:
What I would suggest is to have it as top?: boolean.

So by default, the axis is at the bottom (like an normal chart).

Then, this layer will use the boolean to set the verticalLinePosition.
top=true -> VerticalLinePosition.BOTTOM
else -> VerticalLinePosition.TOP